<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>391</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>802</width>
    <height>391</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPlainTextEdit" name="input"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="output"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>781</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="encode_button">
       <property name="text">
        <string>encode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decode_button">
       <property name="text">
        <string>decode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_button">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>key:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="key">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>29</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="iv_label">
         <property name="text">
          <string>iv:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="iv">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>29</height>
          </size>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="packet_label">
         <property name="text">
          <string>packet method:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="packet_box">
         <item>
          <property name="text">
           <string>ECB(default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CBC</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CFB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OFB</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="padding_label">
         <property name="text">
          <string>padding method:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="padding_box">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Zero(default)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pkcs7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ISO10126</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ANSIX923</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>input</tabstop>
  <tabstop>output</tabstop>
  <tabstop>key</tabstop>
  <tabstop>iv</tabstop>
  <tabstop>packet_box</tabstop>
  <tabstop>padding_box</tabstop>
  <tabstop>decode_button</tabstop>
  <tabstop>clear_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
